packages/template/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@rnx-kit/template",
```

We don't want to publish the template package:
```suggestion
{
  "private": true,
  "name": "@rnx-kit/template",
```

---

If you set this package to private, you can also remove the change file. Beachball ignores private packages.